MySaver/ShopView/ShopsView.cs

```diff
@@ -0,0 +1,46 @@
+﻿using MySaver.Controls;
+using MySaver.Services;
+using System.Collections.ObjectModel;
+
+namespace MySaver.ShopView
+{
+    public partial class ShopsView : BaseView
+    {
+        public ObservableCollection<Shop> Shops { get; } = new();
+        public Command GetShopsCommand { get; }
+        ShopService shopService;
+        public ShopsView(ShopService shopService)
+        {
+            this.shopService = shopService;
+            GetShopsCommand = new Command(async () => await GetShopsAsync());
+        }
+
+        async Task GetShopsAsync()
+        {
+            if (IsBusy)
+                return;
+
+            try
+            {
+                IsBusy = true;
+                var shops = await shopService.GetShops();
+
+                if (Shops.Count != 0)
+                    Shops.Clear();
+
+                foreach (var shop in shops)
+                    Shops.Add(shop);
```

Comment:
The code here is a bit confusing since Shops, shops and shop are all different things, could try to use different names, maybe tempShops for the shops variable